controller/controller.php

```diff
@@ -6,11 +6,18 @@
 use PHPMailer\PHPMailer\OAuth;
 // Alias the League Google OAuth2 provider class
 use League\OAuth2\Client\Provider\Google;
+// Import Emilie Schott classes into the global namespace
+use EmilieSchott\BlogPHP\Model\PostManager; 
 
-function homePage($twig) {
-    echo $twig->render('homeView.twig');
+// Load Postmanager class
```

Comment:
See global comment on comments.
Here, the comment doesn't provide more information compared to the code. The code is clear, we don't need the comment to understand what it does.

Reply:
Ok, I understand better now, thank you. I missed the point.